release_notes/topics/4_0_0_final.adoc

```diff
@@ -0,0 +1,65 @@
+ifeval::[{project_community}==true]
+= 4.0.0.Final
+endif::[]
+ifeval::[{project_product}==true]
+= 7.3.0 CD1
+endif::[]
+
+
+== Authorization Services
+
+=== UMA 2.0
+
+Authorization Services have now introduced support for UMA 2.0 and added support for users to manage user access through
+the account management console. There's also a number of other additions and improvements to authorization services.
+
+=== Pushed Claims
+
+With pushed claims it is now possible for clients to push additional claims to have them used by policies when evaluating
+permissions.
+
+=== Resource Attributes
+
+It is now possible to define attributes on resources in order to have them used by policies when evaluating permissions.
+
+
+== Themes and Theme Resources
+
+It's now possible to hot-deploy themes to Keycloak through a regular provider deployment. We've also added support for
+theme resources. Theme resources allows adding additional templates and resources without creating a theme. Perfect for
```

Comment:
We try not to use contractions in documentation, so if possible, please change the start to "It is now..."